app/models/common/confim.js

```diff
@@ -33,10 +37,15 @@ function ViewModel (details, backNav, errors) {
         }
       ]
     },
-    confirmText: `Are you sure you want to ${details.action} ${details.recordTypeText} record ${details.confirmReferenceText}?`,
+    confirmText: details.confirmText ?? `Are you sure you want to ${details.action} ${details.recordTypeText} record ${details.confirmReferenceText}?`,
     nameOrReference: details.nameOrReference,
     nameOrReferenceText: details.nameOrReferenceText,
     pk: details.pk,
+    inputReference,
```

Comment:
I think this is ok but it's a bit of a hybrid (combining inputReference as well as keeping nameOrReference and nameOrReferenceText. If we had more time, maybe we could align to one method if doing it?